lib/plugins/output-filter/kubernetes-enrichment.js

```diff
@@ -90,6 +90,32 @@ function checkLogsEnabled (pod, data, context) {
   }
 }
 
+function parseReceiverList (receivers) {
```

Comment:
This is exactly the same function as in the `lib/plugins/input/docker/dockerInspect.js` file. Can you put it in a dedicated file and include it where needed. This way we won't have code duplication.

Reply:
Done